API.md

```diff
@@ -285,3 +321,39 @@ If multiple added/omits are present, the <code>add/omit</code> symbol is only pr
     </tr>  </tbody>
 </table>
 
+<a name="AltIntervals"></a>
+
+## AltIntervals : <code>Object</code>
+Intervals affected by the Alt modifier when parsing an altered chord written "C7alt", for example.
+
+**Kind**: global typedef  
+**Properties**
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th><th>Type</th><th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+<tr>
+    <td>fifthFlat</td><td><code>Boolean</code></td><td><p>if the alt modifier should flatten the fifth</p>
```

Comment:
also, why `fifthFlat` and not `flatFifth` or `flatFive`? They seem more natural to me. (opinion here: I would opt for flatFive because it is most likely the term musicians use and hear the most)

Same for others eg 

- sharpFive
- flatNine
- sharpNine
- sharpEleven
- flatThirteen

Reply:
I somehow agree... but for consistency reasons with the rest of the code I'll keep it that way for now. The target audience is the "techie" musician so I think that's fine.
Rationale behind is I wanted the object properties "sorted" by degree then alteration (I can get a bit OCD sometimes)